client/src/components/atoms/Form/InputDate.module.scss

```diff
@@ -39,7 +39,9 @@
     max-width: 16rem;
 }
 
-:global .react-datepicker-popper[data-placement^='top'] .react-datepicker__triangle:before,
+:global
+    .react-datepicker-popper[data-placement^='top']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)